android/assets/jsons/translations/Simplified_Chinese.properties

```
@@ -15,13 +15,11 @@ StartWithCapitalLetter = false
 
 # Documentation: https://f-droid.org/en/docs/Build_Metadata_Reference/#Summary
 # English to translate: https://github.com/yairm210/Unciv/blob/master/fastlane/metadata/android/en-US/short_description.txt
- # Requires translation!
-Fastlane_short_description = 
+Fastlane_short_description = 快捷地简单介绍
 
 # Documentation: https://f-droid.org/en/docs/Build_Metadata_Reference/#Description
 # English to translate: https://github.com/yairm210/Unciv/blob/master/fastlane/metadata/android/en-US/full_description.txt
- # Requires translation!
-Fastlane_full_description = 
+Fastlane_full_description = 完整的介绍内容
```

```suggestion
Fastlane_full_description = 
```
Check the # part above. You can leave it blank for now